hack/internal/tools/yaml-processor/main.go

```diff
@@ -32,7 +34,7 @@ func main() {
 		log.Fatalf("Failed to initialize logger: %v", err)
 	}
 	defer func() {
-		if err := logger.Sync(); err != nil {
+		if err := logger.Sync(); err != nil && !errors.Is(err, syscall.ENOTTY) {
```

Comment:
please add a comment explaining why this particular error is silenced. best if you also post some reference why this is specific enough